### DISCUSSION TITLE
Specifying mavenLocal() in android native librabry build gradle does not seem to work

### DISCUSSION THREAD

**Discussion by visuall** [callstack/react-native-builder-bob#527] [Q&A]
I would like to reference a native Android library from the created React Native library that is installed in the local maven repository. I have added 

allprojects {
    repositories {
        mavenLocal()
        mavenCentral()
        google()
    }
}

at the top of build.gradle that is located inside the android library folder but does not seem to work. The same approach works with normal react app. Am I doing it wrong ?

**Comment by BarbieriMatheus** [comment]
I have the same issue.

**Comment by satya164** [comment]
Libraries can't specify repositories for the app. You need to specify them in the app's `build.gradle`.
